pkg/server/server.go

```diff
@@ -17,23 +17,48 @@ import (
 	"github.com/moov-io/metro2/pkg/utils"
 )
 
-func parseInputFromRequest(r *http.Request) (file.File, error) {
+func parseInputFromRequest(r *http.Request) (file.File, string, error) {
+	if r.Header.Get("Content-Type") == "application/json" {
+		var body struct {
+			Format string                 `json:"format"`
+			Data   map[string]interface{} `json:"data"`
+		}
+
+		decoder := json.NewDecoder(r.Body)
+		if err := decoder.Decode(&body); err != nil {
+			return nil, "", err
+		}
+		defer r.Body.Close()
+
+		str, err := json.Marshal(body.Data)
+		if err != nil {
+			return nil, "", err
+		}
+
+		mf, err := file.CreateFile(str)
+		if err != nil {
+			return nil, "", err
+		}
+
+		return mf, body.Format, nil
```

Comment:
I think we need to do some validation here? It looks like `metro` or `json`. 

https://github.com/moov-io/metro2/blob/f56a3fba5e8240602f05fe5ff378226a7b2b885d/pkg/server/server.go#L80

https://github.com/moov-io/metro2/blob/f56a3fba5e8240602f05fe5ff378226a7b2b885d/pkg/utils/constants.go#L23